### PR TITLE
The quarkus-bootstrap-maven-plugin is listed twice on all the runtime extensions POMs

### LINKED ISSUE #1390: The quarkus-bootstrap-maven-plugin is listed twice on all the runtime extensions POMs
Seems a side effect of https://github.com/apache/camel-quarkus/commit/49f63901777d27f7874a10dfd54006d8f47685e3

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #1390

[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: extensions-core/caffeine-lrucache/runtime/pom.xml [hunk 1]
```diff
@@ -52,39 +52,21 @@
...
-                        <id>update-extension-doc-page</id>
-                        <goals><goal>update-extension-doc-page</goal></goals>
-                        <phase>process-classes</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
-            </plugin>
```
Comment (ppalaga) [inline extensions-core/caffeine-lrucache/runtime/pom.xml:69]: Removing the dupicate quarkus-bootstrap-maven-plugin would be fine, but why are you removing also `camel-quarkus-package-maven-plugin`?

Comment (lburgazzoli) [reply extensions-core/caffeine-lrucache/runtime/pom.xml:69]: next round

Comment (lburgazzoli) [reply extensions-core/caffeine-lrucache/runtime/pom.xml:69]: got confused too, it is still there

Comment (ppalaga) [reply extensions-core/caffeine-lrucache/runtime/pom.xml:69]: Indeed, sorry for the confusion.

Comment (lburgazzoli) [reply extensions-core/caffeine-lrucache/runtime/pom.xml:69]: no worries

**Pull request merged**